project-security/verifying-signatures.md

```diff
@@ -595,10 +595,13 @@ first.](#3-verify-your-qubes-iso))
 ### Why am I getting "WARNING: This key is not certified with a trusted
 signature! There is no indication that the signature belongs to the owner."?
 
-Either you don't have the [Qubes Master Signing
-Key](#1-get-the-qubes-master-signing-key-and-verify-its-authenticity), or you
-didn't [set its trust level
-correctly](#1-get-the-qubes-master-signing-key-and-verify-its-authenticity).
+Several possibilities:
+* you don't have the [Qubes Master Signing
+Key](#1-get-the-qubes-master-signing-key-and-verify-its-authenticity)
+* you didn't [set its trust level
+correctly](#1-get-the-qubes-master-signing-key-and-verify-its-authenticity)
+* in the case of verifying a Git commit or tag, you haven't yet chosen to
+place ultimate PGP trust in the individual signer's key
```

Comment:
The third possibility you just added is actually the same as the second possibility that was already present. Placing ultimate trust in a key is the same as setting the trust level of that key (to "ultimate").

I'm guessing this wasn't clear (or else you wouldn't have been motivated to add the third reason), so perhaps we just need to clarify that existing second reason.

Reply:
My experience:

1. I **did** have the QMSK
2. I **did** set its (meaning the QMSK's) trust level correctly to "ultimate"
3. when verifying a Git commit by Andrew, before giving Andrew's key explicit "ultimate" trust, I got the warning

So in this sequence, the second reason wasn't the cause for the warning I received.  It was the third reason.  Ultimately trusting your doc signing key eliminated the warning for me.

Reply:
It appears that's because my doc signing key is not signed by the QMSK (nor are any of the other doc signing keys). I guess that's our policy. Only some of the keys in the secpack are signed by the QMSK.

I'll document this as part of the revamp I'm doing for this page.